solar_system.rb

```diff
@@ -0,0 +1,262 @@
+# Create a Planet class which will represent a planet.
+# Add an initialize method which takes several arguments and uses them to set the class' instance variables.
+# Create a method that returns the Planet's attributes in an easy to read fashion.
+# Create reader methods to give a user access to read the instance variables.
+# Make your SolarSystem class take an array of Planets instead of hashes.
+
+# Verify user input for integers
+def verify(user_input)
+  until user_input.to_i.to_s == user_input
```

Comment:
I like that you've extracted this logic into a couple of methods - makes the driver code below concise and readable.